Move Single Asset Vault docs from opensource (#3261)
XRPLF/xrpl-dev-portal: docs/references/http-websocket-apis/public-api-methods/ledger-methods/ledger_entry.md

{% try-it method="ledger_entry-mptoken" /%}

### Get Vault Entry

> Retrieve a `Vault` object from the ledger. This is similar to the [vault_info method][], but the `ledger_entry` version returns only the ledger entry as stored.

Reviewer: ```suggestion
Retrieve a `Vault` entry from the ledger. This is similar to the [vault_info method][], but the `ledger_entry` version doesn't return information about vault shares.
```

{% amendment-disclaimer name="SingleAssetVault" /%}

| Field        | Type             | Description           |
|:-------------|:-----------------|:----------------------|
| `vault`      | String           | The [ledger entry ID](../../../protocol/ledger-data/common-fields.md#ledger-entry-id) of a [Vault](../../../protocol/ledger-data/ledger-entry-types/vault.md) object to retrieve. |

{% tabs %}

{% tab label="WebSocket" %}
```json
{
    "id": "example_get_vault_entry",
    "command": "ledger_entry",
    "vault": "45E6742527EDE6A2B537AE8A77B8D8CCFEFE115A22B3BF664A39407631F9A166",
    "ledger_index": "validated"
}
```
{% /tab %}

{% tab label="JSON-RPC" %}
```json
{
    "method": "ledger_entry",
    "params": [
        {
            "vault": "45E6742527EDE6A2B537AE8A77B8D8CCFEFE115A22B3BF664A39407631F9A166",
            "ledger_index": "validated"
        }
    ]
}
```
{% /tab %}

{% tab label="Commandline" %}
```sh
rippled json ledger_entry '{ "vault": "45E6742527EDE6A2B537AE8A77B8D8CCFEFE115A22B3BF664A39407631F9A166", "ledger_index": "validated" }'
```
{% /tab %}

{% /tabs %}

<!-- TODO: Add when deployed to Devnet/Testnet -->

> <!-- {% try-it method="ledger_entry-single-asset-vault" server="testnet" /%} -->

Reviewer: Don't forget to add the try-it button to devnet examples.
